artiq/firmware/libboard_artiq/hmc830_7043.rs

```diff
@@ -113,18 +80,66 @@ mod hmc830 {
     }
 
     pub fn init() -> Result<(), &'static str> {
+        // Configure HMC830 for integer-N operation
+        // See "PLLs with integrated VCO- RF Applications Product & Operating
+        // Guide"
         spi_setup();
-        info!("loading configuration...");
-        for &(addr, data) in HMC830_WRITES.iter() {
-            write(addr, data);
-        }
+        info!("loading HMC830 configuration...");
+
+        write(0x0, 0x20);    // software reset
+        write(0x0, 0x00);    // normal operation
+        write(0x6, 0x307ca); // integer-N mode (NB data sheet table 5.8 not self-consistent)
+        write(0x7, 0x4d);    // digital lock detect, 1/2 cycle window (6.5ns window)
+        write(0x9, 0x2850);  // charge pump: 1.6mA, no offset
+        write(0xa, 0x2045);  // for wideband devices like the HMC830
```

Comment:
I guess you are OK with not doing a review of the values, right?

Reply:
Extra reviews are always welcome. However, in this case I don't think it's necessary. I've checked this all through against the data sheet carefully and tested it on my hardware. I'm inclined to say that if it works reliably on other HW as well then further review isn't worth the time it would take (but if you have spare time then go for it!).